tests/plugins/os/unix/bsd/darwin/ios/test__os.py

```diff
@@ -0,0 +1,54 @@
+from dissect.target.filesystem import VirtualFilesystem
+from dissect.target.plugin import OperatingSystem
+from dissect.target.plugins.os.unix.bsd.darwin.ios._os import IOSPlugin
+from dissect.target.plugins.os.unix.bsd.darwin.ios.applications import (
+    IOSApplicationsPlugin,
+)
+from dissect.target.plugins.os.unix.bsd.darwin.ios.locale import LocalePlugin
+from dissect.target.target import Target
+from tests._utils import absolute_path
+
+
+def test_ios_detect(target_bare: Target, fs_ios: VirtualFilesystem) -> None:
+    """Test if we correctly detect an iOS target."""
+    target_bare.filesystems.add(fs_ios)
+    target_bare.apply()
+    assert IOSPlugin.detect(target_bare)
+    assert target_bare._os_plugin is IOSPlugin
+
+
+def test_ios_os(target_ios: Target, fs_ios: VirtualFilesystem) -> None:
+    """Test if the iOS plugin works on an iOS target.
+
+    Data based on example iOS image from Digital Corpora.
+
+    Resources:
+        - https://corp.digitalcorpora.org/corpora/mobile/iOS17/
+    """
+
+    fs_ios.map_dir("/", absolute_path("_data/plugins/os/unix/bsd/darwin/ios/_os"))
+    target_ios.add_plugin(IOSPlugin)
+    target_ios.add_plugin(LocalePlugin)
+    target_ios.add_plugin(IOSApplicationsPlugin)
```

Comment:
```suggestion
    target_ios.add_plugin(IOSApplicationsPlugin)
    target_ios.apply()
```
This properly initializes the OS portion of the target and allows the `.language` to work.